webapp/src/components/properties/link/link.scss

```diff
@@ -0,0 +1,26 @@
+.property-link {
+    &.url {
+        padding-right: 2rem;
+    }
+
+    a {
+        padding: 8px; // increases clickable area for better UX
+        display: none;
+    }
+
+    a, a:visited {
+        color: rgb(var(--body-color));
+    }
+
+    a:hover {
+        background:  unset;
+    }
+
+    &:hover {
+        a {
+            display: inline;
+        }
+    }
```

Comment:
If we remove display: none, we can remove this.

Reply:
Keeping the icon hidden until hovered on, so need to preserve this.